mmv1/templates/terraform/examples/dns_managed_zone_cloud_logging.tf.erb

```diff
@@ -0,0 +1,14 @@
+# [START dns_managed_zone_cloud_logging]
```

Comment:
```suggestion
```
these comments `# [START` and `# [END` still appear in some examples, but they no longer have any purpose. We can delete them from here

Reply:
Thanks for pointing it out. Removed these comments.